settings.gradle

```diff
@@ -2,8 +2,6 @@ rootProject.name = 'MekHQRoot'
 
 include 'MekHQ'
 
-enableFeaturePreview('STABLE_PUBLISHING')
```

Comment:
If memory serves, there was going to be a change in the way publishing tasks are handled in 5.0 (which wasn't released yet), and adding this allowed using it with 4.9 so it wouldn't have to be rewritten if we switch to a later version. So it served its purpose.